python-avd/pyavd/_utils/password_utils/password.py

```diff
@@ -335,6 +335,48 @@ def isis_decrypt(password: str, key: str, mode: str) -> str:
         raise ValueError(msg) from exc
 
 
+########
+# Tacacs
+########
+def tacacs_encrypt(password: str, salt: Literal[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]) -> str:
+    """
+    Encrypt (obfuscate) a password from insecure type-7.
+
+    Args:
+        password: The clear text password
+        salt: A number within the range 0-15.
+
+    Returns:
+        str: The decrypted password as a string.
```

Comment:
```suggestion
        str: The encrypted password as a string.
```
The return will be encrypted.